CONTRIBUTING.md

```diff
@@ -1018,6 +1026,22 @@ You can interact with the running kubernetes clusters now using `kubectl [COMMAN
 
 * `kind delete clusters --all` is helpful for cleanup!
 
+2. The other option is to use the helper method in the framework: `helpers.WaitForInput(t)` at the spot in your acceptance test where you would like to pause execution to inspect the cluster. This will pause the test execution until you execute a request to `localhost:8715` which tells the test to continue running, you can override the port value used by setting the `CONSUL_K8S_TEST_PAUSE_PORT` environment variable to a port of your choosing. When running the tests with the `-v` flag you will see a log output of the endpoint that the test is waiting on.
+
+```go
+
+import "github.com/hashicorp/consul-k8s/acceptance/framework/helpers"
+
+func TestSomeTest(t *testing.T) {
```

Comment:
~ Looks like a mix of tabs and spaces here. Probably fixed by most IDEs on paste but could shore up now